README.md

```diff
@@ -8,8 +8,8 @@
 
 <p align="center">
   <a href="https://github.com/input-output-hk/cardano-wallet/releases"><img src="https://img.shields.io/github/release-pre/input-output-hk/cardano-wallet.svg?style=for-the-badge"  /></a>
-  <a href="https://buildkite.com/input-output-hk/cardano-wallet"><img src="https://img.shields.io/buildkite/831447d2d2f363c702fae34fc7519c5df5aeb1115af0a7f42b/master?label=BORS&style=for-the-badge"/></a>  
```

Comment:
This was the `pre-merge` pipeline being labeled as BORS https://buildkite.com/input-output-hk/cardano-wallet-pre-merge/settings/badges